package.js

```diff
@@ -2,15 +2,20 @@
 
 Package.describe({
   name: 'twbs:bootstrap', // https://atmospherejs.com/twbs/bootstrap
-  summary: 'The most popular front-end framework for developing responsive, mobile first projects on the web.',
+  summary:
+    'The most popular front-end framework for developing responsive, mobile first projects on the web.',
   version: '5.0.0-alpha1',
   git: 'https://github.com/twbs/bootstrap.git'
 });
 
 Package.onUse(function (api) {
   api.versionsFrom('METEOR@1.0');
-  api.addFiles([
-    'dist/css/bootstrap.css',
-    'dist/js/bootstrap.js'
-  ], 'client');
+  api.addFiles(
```

Comment:
I removed the `api.use('jquery');` from the version of `package.js` on the `master`, since v5.0.0-alpha1 does not use jQuery.